README.md

```diff
@@ -90,6 +90,42 @@ public class Main {
 }
 ```
 
+##### Per query metrics
```

Comment:
I think we also need this documentation for Scala :)

Reply:
yep, thanks for this one :)